### PR TITLE
Open link

### PR DESCRIPTION
**Pull request by nchevobbe**
Associated Issue: #4049

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes

* Add an `openLink` function in assets/panel.js that open a link in a new tab (as well as a version for the launchpad).
* Create an action that call this openLink function
* Call the action in components that declare `ObjectInspector` or `Reps` component

### Test Plan

- [x] Add `"http://mozilla.org"` as a watch expression, click on the resulting link

---

This is working in launchpad and in toolbox. I am not sure that this should be in an action though, but I didn't want to change too much things to pass it as a prop to the App components and pipe it down.  One thing to have in mind is that we will also need the toolbox integration for the open-in-inspector button on DomElement reps. So maybe all the toolbox integration can be done in the actions/toolbox.js file.  I plan to add a mochitest to make sure we don't regress on this in the different components.

Do tell me if it's not appropriate to have an action that does not dispatch anything (I feel weird about it) and if we should do this in another way.

By the way if we are in a hurry, the first commit of this PR fixes the bug (but does not open the link, which I think is fine since it's a new feature).

### CODE REVIEW

**Comment by jasonLaster** [comment]
I tested preview and scopes here: https://link-val.glitch.me/

File: src/components/SecondaryPanes/Scopes.js [hunk 1]
```diff
@@ -9,13 +9,26 @@ import {
...
-import { getScopes } from "../../utils/scopes";
 
+import { getScopes } from "../../utils/scopes";
 import { ObjectInspector } from "devtools-reps";
 
 import "./Scopes.css";
 
+import type { Pause, LoadedObject, Frame } from "debugger-html";
+
+type Props = {
```
Comment (nchevobbe) [inline src/components/SecondaryPanes/Scopes.js]: 👍

**Comment by nchevobbe** [comment]
Looks ready to land. I'm working on mochitests to make sure we don't regress.

CI is showing failure:
```
failed test:  devtools/client/debugger/new/test/mochitest/browser_dbg_keyboard_navigation.js 
failed test:  devtools/client/debugger/new/test/mochitest/browser_dbg-wasm-sourcemaps.js 
```

but I don't think there relevant to this PR ?

**Comment by jasonLaster** [comment]
yeah, wasm is a new test so it could be flakey

**Pull request merged**